src/coreclr/nativeaot/System.Private.CoreLib/src/System/Enum.CoreRT.cs

```diff
@@ -222,7 +222,7 @@ public static string[] GetNames(Type enumType)
                 return enumType.GetEnumNames();
 
             if (!enumType.IsEnum)
-                throw new ArgumentException(SR.Arg_MustBeEnum);
+                throw new ArgumentException(SR.Arg_MustBeEnum, nameof(enumType));
```

Comment:
Oh well. My local repo is a mess so I would prefer this is just taken for the ride.